.devcontainer/Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM ubuntu:20.04
```

Comment:
I think this Dockerfile should be project's default Dockerfile in the root. I don't think Codespaces requires its own container. If Git and Python is present things should work ok.